lib/src/web_driver.dart

```diff
@@ -141,9 +141,18 @@ class WebDriver implements SearchContext {
   Mouse get mouse => new Mouse._(this);
 
   /// Take a screenshot of the current page as PNG.
```

Comment:
Can you update the doc comments of all three methods? It should outline the differences between the three screenshot methods.